### ISSUE TITLE
Automated build for all tags only works when given Full Access

### ISSUE THREAD

**Issue by xy2z** [docker/hub-feedback#1546]
**TL;DR**
Only `latest` tag is built on automated builds with **limited access** (read) to GitHub.
When I give "Full Access" to my GitHub account, my tags are being built as expected.

**Steps to reproduce**
1. Setup an automated build to a GitHub repo with **limited access** (read).
1. Setup the webhook in the GitHub repo, using "Let me select individual events.". Select "Branch or tag creation" and "Pushes".
1. Docker Hub: Make sure all tags are targeted by adding "Type: Tag" and click save (this is default enabled)
1. Make a change in your GitHub repo, tag it and push it.
1. See Docker Hub: Build Details. Only the "latest" tag is built.

Sometimes the latest tag was built 2-3 times, so it may be GitHub sending the wrong information.
I've tried with Content type: application/json and application/x-www-form-urlencoded, it didn't matter.

**Solution**
Give Docker Hub **Full Access** (read/write) to your GitHub account, and your tag is being built on the next push.

**Comment by PeterDaveHello** [comment]
I got the same problem even with full access.
